### ISSUE TITLE
Feature: smaller window for audio only

### ISSUE THREAD

**Issue by dolence** [luvolondon/fvtt-module-jitsiwebrtc#18]
Would be very nice having smaller chat windows when not using video. There is no reason to have a big portion of screen covered by an video overlay that isn't being streamed.

**Comment by bekriebel** [comment]
I agree with this. However, I plan to put this into my [WebRTC Tweaks](https://github.com/bekriebel/fvtt-module-webrtc_tweaks) module since this applies to both the stock EasyRTC implementation and Jitsi.

**Comment by dolence** [comment]
Thank you, @bekriebel
